.github/workflows/update-search.yml

```
@@ -0,0 +1,44 @@
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0  # Fetch all history for proper git operations
+    
+    - name: Clone comphy-search repository
+      run: |
+        git clone https://github.com/comphy-lab/comphy-search.git
+        mkdir -p docs/assets/js
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)